src/localization/locales/en-US.json

```diff
@@ -119,7 +119,9 @@
         "members_list_pending_confirmation_message": "Do you want to deny {{userName}}’s request for membership in {{name}}?",
         "members_list_pending_confirmation_button": "Deny Request",
         "members_list_pending_reject": "Deny",
-        "home_pending_message": "Waiting for the group manager’s approval"
+        "home_pending_message": "Waiting for the group manager’s approval",
+        "breadcrumbs_view": "{{groupName}}",
```

Comment:
question: if this is just a variable, do we need a localized string for it?

Reply:
@paulschreiber This side effect supports the flexibility to define labels for each page that needs to have breadcrumb without having manage a complex structure for each one of the pages